app/views/content/what-is-a-pgce.md

```diff
@@ -51,6 +51,8 @@ Getting QTS only means that you’re qualified to teach in most schools in Engla
 
```

Comment:
We don't talk about QTS with PGCE anywhere in this section @EllieNodder - should we?